scripts/build-android.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Don't know if we use bash specific stuff (guessing probably), but it'd be nice if this was `#!/usr/bin/env sh`

Reply:
It's used to access `current dir`. There might be better way to do this, not expert enough.
Note it only requires you to have `bash` available, not to use it (I use `zsh`).